servers/rendering/renderer_rd/renderer_canvas_render_rd.cpp

```diff
@@ -957,7 +957,48 @@ void RendererCanvasRenderRD::_render_item(RD::DrawListID p_draw_list, RID p_rend
 
 		c = c->next;
 	}
+#ifdef DEBUG_ENABLED
+	if (debug_redraw && p_item->debug_redraw_time > 0.0) {
+		Color dc = debug_redraw_color;
+		dc.a *= p_item->debug_redraw_time / debug_redraw_time;
 
+		RID pipeline = pipeline_variants->variants[PIPELINE_LIGHT_MODE_DISABLED][PIPELINE_VARIANT_QUAD].get_render_pipeline(RD::INVALID_ID, p_framebuffer_format);
+		RD::get_singleton()->draw_list_bind_render_pipeline(p_draw_list, pipeline);
+
+		//bind textures
```

Comment:
```suggestion
		// Bind textures.
```

Reply:
I saw this, but decided not to change it. The whole file needs a cleanup pass:
```
servers/rendering/renderer_rd/renderer_canvas_render_rd.cpp:				//bind pipeline
servers/rendering/renderer_rd/renderer_canvas_render_rd.cpp:				//bind textures
servers/rendering/renderer_rd/renderer_canvas_render_rd.cpp:				//bind pipeline
servers/rendering/renderer_rd/renderer_canvas_render_rd.cpp:				//bind textures
servers/rendering/renderer_rd/renderer_canvas_render_rd.cpp:				//bind pipeline
servers/rendering/renderer_rd/renderer_canvas_render_rd.cpp:				//bind textures
servers/rendering/renderer_rd/renderer_canvas_render_rd.cpp:				//bind pipeline
servers/rendering/renderer_rd/renderer_canvas_render_rd.cpp:				//bind textures
```